tv_apps/fling-player/test/unit/mock_video_element.js

```diff
@@ -0,0 +1,66 @@
+(function(exports) {
```

Comment:
Please add a comment like "XXX workaround bug <#>, should remove this file once we fix it.".